### ISSUE TITLE
Error when dev

### ISSUE THREAD

**Issue by erguotou520** [pastahito/electron-vue-webpack#1]
error when run `npm run dev`
```
ERROR in ./app/src/App.vue
Module build failed: TypeError: this._init is not a function
    at Object.Vue$2 (/path/electron-vue-webpack/node_modules/.2.1.4@vue/dist/vue.runtime.common.js:3318:8)
 @ ./app/src/entry.js 2:0-27
```

**Comment by hifizz** [comment]
@erguotou520 I get this Error too. But I has get the answer from this issue:[Module build failed: TypeError: this._init is not a function](https://github.com/vuejs/vue-loader/issues/409).But in this project,there are two errors.

Fix the bug is easy.just change the code

```js
{
                test: /\.vue$/,
                loader: 'vue'
            },
            {
                test: /\.(png|jpg|gif|svg)$/,
                loader: 'file',
                query: {
                    name: '[name].[ext]?[hash]'
                }
            }
```

to 

```js
{
                test: /\.vue$/,
                loader: 'vue-loader'
            },
            {
                test: /\.(png|jpg|gif|svg)$/,
                loader: 'file-loader',
                query: {
                    name: '[name].[ext]?[hash]'
                }
            }
```

Bingo!

**Comment by pastahito** [comment]
Fixed in #2

**Comment by renato-rg** [comment]
Fixed in #2
